gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
+networkTimeout=10000
```

Comment:
Was this generated or intentional? Can you say more if it's the latter?

Reply:
Yeah, that comes automatically from running `./gradlew wrapper`